web/controllers/sitemap_controller.ex

```diff
@@ -0,0 +1,11 @@
+defmodule HexWeb.SitemapController do
+  use HexWeb.Web, :controller
+
+  def sitemap(conn, _params) do
+    packages = Package.all(1, 50000)
+              |> HexWeb.Repo.all
```

Comment:
Do `HexWeb.Repo.all(Package)` instead. This way we don't preload downloads and will fetch all packages.